### DISCUSSION TITLE
Can I use another ORM in Strapi ?, for example Squelize

### DISCUSSION THREAD

**Discussion by ricar2ce** [strapi/strapi#5979] [Legacy Topics]
Hello, i have a question
Can I use another ORM in Strapi ?
I would rather be able to use squalize or Lucid.
anyone know if it can?

**Comment by lauriejim** [comment]
Hello! It's not possible to do that.
This connector doesn't exist for SQL databases.

Can you tell us what are your motivations to have these ORM instead of Knex + Bookshelf?
Thank you.
